companion/src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-PROJECT( companion )
```

Comment:
Why would we have a different project name? It should not be dependant of the release!